File: model/Payment.js
const mongoose = require("mongoose");

const paymentSchema = new mongoose.Schema({
    cardId: {
        type: String,
        require: true,
        unique: true
    },
    fullName: {
        type: String,
        require: true,
        minlength: 6
    },
    nameCard: {
        type: String,
        require: true,
        minlength: 3
    }

}, { timestamps: true });
module.exports = mongoose.model("Payment", paymentSchema);

File: Aiking/Models/Payment.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Aiking.Models
{
    [Table("Payment")]
    public class Payment
    {
        public int Id { get; set; }
        public string? CardId { get; set; }
        public string? FullName { get; set; }
        public string? NameCard { get; set; }
        public DateTime createdAt { get; set; }
        public DateTime updatedAt { get; set; }
        public int UserID { get; set; }
    }
}
